### PR TITLE
Format w/ JuliaWTF/WatchJuliaBurn.jl

### PR DESCRIPTION
**Pull request by suvayu**
Hilarious and anxiety inducing

### CODE REVIEW

**Review by clizbe — COMMENTED** [review]
LMAO I feel like this is actually pretty clear.

File: src/pipeline.jl [hunk 1]
```diff
@@ -67,124 +67,118 @@ end
...
+        ☝️::👍👎 = 👎,
     )
 
 Create a table from a file source (CSV, Parquet, line delimited JSON, etc)
 
 The resulting table is saved as the table `name`.  The name of the
 created table is returned.
 
-Optionally, if `show` is `true`, the table is returned as a Julia
+Optionally, if `☝️` is `✅`, the table is returned as a Julia
```
Comment (clizbe) [inline src/pipeline.jl:81]: Why is this ✅ and not 👍?

File: src/pipeline.jl [hunk 2]
```diff
@@ -67,124 +67,118 @@ end
...
 
 It is also possible to create the table as a temporary table by
 setting the `tmp` flag, i.e. the table is session scoped.  It is
 deleted when you close the connection with DuckDB.
 
-When `show` is `false`, and `name` was not provided, a table name
+When `☝️` is `👎`, and `name` was not provided, a table name
 autotomatically generated from the basename of the filename is used.
-This also unconditionally sets the temporary table flag to `true`.
+This also unconditionally sets the temporary table flag to `👌`.
```
Comment (clizbe) [inline src/pipeline.jl:91]: This should also be 👍

Comment (suvayu) [reply src/pipeline.jl:91]: Good spots! I'm not sure, initially I thought the package does a find replace for certain literal objects, and then does some kind of a heuristic matching with variable names.  But now you found these 2 deviations, so maybe it replaces randomly from a set of "similar" emojis?  I could be convinced that `true` is the same as either of these: :+1: :white_check_mark: :ok_hand:!

:stuck_out_tongue:

Comment (clizbe) [reply src/pipeline.jl:91]: Even better!

**Pull request closed without merging**